apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx

```diff
@@ -150,6 +152,12 @@ function ReceiveConfirmationInner({ navigation, route, account, parentAccount }:
     }
   }, [currency, route.params?.createTokenAccount, mainAccount, dispatch, hasAddedTokenAccount]);
 
+  const completeAction = useCompleteActionCallback();
+
+  useEffect(() => {
+    completeAction(PostOnboardingActionId.assetsTransfer);
+  }, [completeAction]);
```

Comment:
Same question as other useEffect do we want to call this only onMount or whenever completeAction changes too ?

Reply:
I'll update this one to make sure we do not call this one again in case the completeAction changes